schemas/ui/v1.0/ui.schema

```diff
@@ -182,6 +187,24 @@
             },
             "additionalProperties": false
         },
+        "FlowUIOptions": {
+            "type": "object",
+            "properties": {
+                "widget": {
+                    "title": "Flow Widget Name",
+                    "description": "Flow widget name of current $kind.",
+                    "type": "string",
```

Comment:
Is this an open-ended string or should it be an enum?

Reply:
The former one, its value is in a set of widget names which can be either built-in widgets (declared as examples) or customized widgets with arbitrary name (provided by ui-plugin as extension).